### PR TITLE
Project Info

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

Updates the protocol buffers and adds the Project Info RPC method to the SDK.

Fixes SC-16531

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [x]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

**Comment by bbengfort** [comment]
NOTE: these tests will fail until https://github.com/rotationalio/ensign/pull/449 is merged.

File: info.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// Info returns summary statistics that describe the state of the project that you can
+// connect to with your API key. Statistics include the number of topics, the number
+// topics that are readonly (a subset of the number of topics) and the number of events.
+// The project for the statistics is determined by the project your API key has access
+// to (API keys are issued to projects). You can also specify a list of topicIDs to get
+// the statistics for (e.g. filtering the statistics for one or more topics).
+//
+// TODO: allow users to specify either topic names or topic IDs.
+func (c *Client) Info(ctx context.Context, topicIDs ...string) (info *api.ProjectInfo, err error) {
```
Comment (pdeziel) [inline info.go:19]: Makes sense to me, have you created a story to add this to pyensign?

**Review by pdeziel — APPROVED** [review]
Contingent on the tests passing this looks good to me!

File: info.go (hunk 1 above)
Comment (bbengfort) [reply info.go:19]: Just created SC-16569 for this.

**Pull request merged**